core/src/main/scala/epollcat/internal/ch/socket.scala

```diff
@@ -18,14 +18,14 @@ package epollcat.internal.ch
 
 import scala.annotation.nowarn
 import scala.scalanative.unsafe._
+import scala.scalanative.posix.sys.socket._
 
 @extern
 @nowarn
 private[ch] object socket {
   final val SOCK_NONBLOCK = 2048 // only in Linux and FreeBSD, but not macOS
 
-  def accept(sockfd: CInt, addr: Ptr[Byte], addrlen: Ptr[Byte]): CInt = extern
```

Comment:
Since we are no longer calling `accept` with `null` arguments, we no longer need our own `extern`.